history/taskana-simplehistory-rest-spring/src/test/java/pro/taskana/simplehistory/rest/TaskHistoryEventControllerIntTest.java

```diff
@@ -31,6 +31,14 @@
 @TaskanaSpringBootTest
 class TaskHistoryEventControllerIntTest {
 
+  private static final ParameterizedTypeReference<TaskHistoryEventPagedRepresentationModel>
+      TASK_HISTORY_EVENT_PAGED_REPRESENTATION_MODEL_TYPE =
+          new ParameterizedTypeReference<TaskHistoryEventPagedRepresentationModel>() {};
+
+  private static final ParameterizedTypeReference<TaskHistoryEventRepresentationModel>
+      TASK_HISTORY_EVENT_REPRESENTATION_MODEL_TYPE =
+          new ParameterizedTypeReference<TaskHistoryEventRepresentationModel>() {};
```

Comment:
Dang! I don't like this Syntax. We have to talk about this in our cop. 
This is just a reminder for me to add the entry.